ansible/roles/vm_set/library/vm_topology.py

```diff
@@ -426,6 +426,20 @@ def destroy_bridges(self):
             self.destroy_ovs_bridge(VS_CHASSIS_INBAND_BRIDGE_NAME)
             self.destroy_ovs_bridge(VS_CHASSIS_MIDPLANE_BRIDGE_NAME)
 
+    def reinstall_8021q(self):
```

Comment:
reinstall is not the right name, reinsert is better for this operation.

Reply:
Hi @yxieca , updated the commit according your comments.